base/pkg/entry.jl

```diff
@@ -51,7 +51,7 @@ function add(pkg::AbstractString, vers::VersionSet)
     @sync begin
         @async if !edit(Reqs.add,pkg,vers)
             ispath(pkg) || throw(PkgError("unknown package $pkg"))
-            info("Nothing to be done")
+            info("`$pkg` package is already installed")
```

Comment:
For consistency with the other strings here we should probably do without the backquotes. For example, see line 53.